cloud/jenkins/psmdb_operator_lke_version.groovy

```diff
@@ -1,5 +1,9 @@
 
 void CreateCluster(String CLUSTER_PREFIX) {
+    if ( "${params.CLUSTER_WIDE}" == "YES" ) {
+        env.OPERATOR_NS = 'psmdb-operator'
+    }
+
     sh '''
         retry() {
```

Comment:
Can we delete it? I mean this task

Reply:
I think yes, but @cap1984 and @hors added these so I will leave it up to them.